src/main/kotlin/no/nav/familie/ef/sak/behandling/grunnbelop/GOmregningTaskService.kt

```diff
@@ -4,23 +4,30 @@ import no.nav.familie.ef.sak.beregning.nyesteGrunnbeløpGyldigFraOgMed
 import no.nav.familie.ef.sak.fagsak.FagsakRepository
 import no.nav.familie.ef.sak.infrastruktur.exception.feilHvisIkke
 import no.nav.familie.ef.sak.infrastruktur.featuretoggle.FeatureToggleService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
 import org.springframework.stereotype.Service
 
 @Service
 class GOmregningTaskService(val fagsakRepository: FagsakRepository,
                             val featureToggleService: FeatureToggleService,
                             val gOmregningTask: GOmregningTask) {
 
+    val logger: Logger = LoggerFactory.getLogger(this::class.java)
```

Comment:
Det går også å skrive med kun `javaClass`
```suggestion
    val logger: Logger = LoggerFactory.getLogger(javaClass)
```